lib/config/decrypt.ts

```diff
@@ -106,33 +107,44 @@ export async function tryDecrypt(
         const { o: org, r: repo, v: value } = decryptedObj.data;
         if (is.nonEmptyString(value)) {
           if (is.nonEmptyString(org)) {
-            const orgName = org.replace(regEx(/\/$/), ''); // Strip trailing slash
+            const orgPrefixes = org
+              .split(',')
+              .map((o) => o.trim())
+              .map((o) => o.toLowerCase())
```

Comment:
we should use upper case for case insensitive compares. we probably should add a string util for it to always reuse.